finesse/gui/stepper_motor_view.py

```diff
@@ -0,0 +1,59 @@
+"""Code for controlling the stepper motor which moves the mirror."""
+from pubsub import pub
+from PySide6.QtWidgets import (
+    QButtonGroup,
+    QGridLayout,
+    QGroupBox,
+    QPushButton,
+    QSpinBox,
+)
+
+from ..config import ANGLE_PRESETS
+
+
+class StepperMotorControl(QGroupBox):
+    """A control showing buttons for moving the mirror to a target."""
+
+    def __init__(self) -> None:
+        """Create a new StepperMotorControl."""
+        super().__init__("Target control")
+
+        layout = QGridLayout()
+
+        # Bundle all the buttons for moving the mirror into one group
+        self.button_group = QButtonGroup()
+        self.button_group.buttonClicked.connect(self._preset_clicked)  # type: ignore
+
+        # Add all the buttons for preset positions
+        BUTTONS_PER_ROW = 4
+        for i, preset in enumerate(ANGLE_PRESETS):
+            btn = self._add_checkable_button(preset.upper())
+            self.button_group.addButton(btn)
+
+            row, col = divmod(i, BUTTONS_PER_ROW)
+            layout.addWidget(btn, row, col)
+
+        # We also have a way for users to move the mirror to an angle of their choice
+        self.angle = QSpinBox()
+        self.angle.setMaximum(359)
```

Comment:
This makes the angular resolution equal to 1º. Is that good enough for the application?

Reply:
It should be. I think that's what the old program has.